frontend/app/src/components/account-management/upgrade/UpgradeProgressDisplay.vue

```diff
@@ -51,6 +52,12 @@ const dataMigrationStatusData: ComputedRef<CurrentDbUpgradeProgress | null> =
       description: description || ''
     };
   });
+
+const { clearInternalTokens } = useNewlyDetectedTokens();
+
+onBeforeMount(() => {
+  clearInternalTokens();
```

Comment:
since this, if before mount we can't check for dbUpgradeStatus, really.

Lefteris mentioned that this should not happen if it's only data migration

Reply:
so this is only happens if `dbUpgradeProgressData` is not null?

Reply:
yes exactly

Reply:
updated